twilio/twilio.go

```diff
@@ -105,6 +106,8 @@ func NewClient(accountSID string, authToken string) *Twilio {
 		Client: &client.Client{
 			Credentials: credentials,
 			BaseURL:     "twilio.com",
+			Edge:        os.Getenv("TWILIO_EDGE"),
```

Comment:
Not sure if this difference will change the results, but `Getenv` can return an empty value or an unset value ([docs](https://golang.org/pkg/os/#Getenv)).

Reply:
Yes that is the desired behavior.